tests/integration/targets/mso_schema_site_external_epg/tasks/main.yml

```diff
@@ -666,15 +683,15 @@
       external_epg: ansible_test_2
       l3out: L3out1
       state: query
-    ignore_errors: yes
+    ignore_errors: true
     register: nm_non_existing_site_template
 
   - name: Verify cm_non_existing_site_template and nm_non_existing_site_template
     assert:
       that:
       - cm_non_existing_site_template is not changed
       - nm_non_existing_site_template is not changed
-      - cm_non_existing_site_template.msg == nm_non_existing_site_template.msg == "Provided template 'Template3' does not exist. Existing templates{{':'}} Template1, Template2, Template3"
+      - cm_non_existing_site_template.msg == nm_non_existing_site_template.msg == "Provided site/siteId/template 'ansible_test/6398a4b9f1ca793733c7d609/Template3' does not exist. Existing siteIds/templates{{':'}} 6398a4b9f1ca793733c7d609/Template1, 6398a4b9f1ca793733c7d609/Template2"
```

Comment:
Should we make 6398a4b9f1ca793733c7d609 (siteId) a dynamic value? I assume if sites are reset for some reason that these sites might have new id's which would then break CI.